libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/theme/components/TopAppBar.kt

```diff
@@ -42,21 +45,26 @@ fun TopAppBar(
     actions: @Composable RowScope.() -> Unit = {},
     windowInsets: WindowInsets = TopAppBarDefaults.windowInsets,
     colors: TopAppBarColors = TopAppBarDefaults.topAppBarColors(),
-    scrollBehavior: TopAppBarScrollBehavior? = null
+    scrollBehavior: TopAppBarScrollBehavior? = null,
+    isVisible: Boolean = true,
```

Comment:
This feels odd.

It shouldn't be the responsibility of any composable to provide a way to hide itself.
If you don't want it to be shown you should wrap it with an `if` just outside the point where you call it.

Maybe I didn't understand your intention here.

Reply:
You're right, I will update remove that.